Game elements/game.js

```diff
@@ -14,18 +33,86 @@ class Food {
           this.positionY = Math.random() * 80;
           this.food.style.bottom = this.positionY + 2 +"%";
           this.positionY;
+
      }
 
-     runToLeft() {
-          setInterval(() => {
-               if (this.positionX >= 95) {
-                    this.food.remove();
-               }
-               if (this.positionX <= 100) {
-                    this.food.style.left = this.positionX + "%";
-                    this.positionX += .4;
-               }
-          }, 50);
+     runToRight() {
+
+          this.runningInterval = 
+               setInterval(() => {
+                    if (this.positionX >= 95) {
```

Comment:
Dobrze jest takie liczby, które coś znaczą, wyciagnać do jakiś zmiennych (np pisanych wielkimi literami, zeby sie wyroznialy(, bo ja nie znajac aplikacji nie mam pojecia co to 95, a jak bym zobaczyl cos w stylu START_POSITION to bym moze wiecej skumal